docs/data/versions.json

```diff
@@ -0,0 +1,14 @@
+{
+	"params": {
+		"currentVersion": "2.0",
+		"ltsVersion": "1.0",
+		"previousVersions": ["1.0"],
+		"allVersions": ["1.0", "2.0"]
```

Comment:
I don't _think_ `allVersions` should be necessary. This will always be `previousVersions` + `currentVersion`, right?

Reply:
Removed